### PR TITLE
Upgrade untool babel

### PR DESCRIPTION
**Pull request by robin-drexler**

### CODE REVIEW

File: packages/spec/integration/jest-preset/__tests__/preset.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+describe('jest-preset', () => {
+  it('allows to use typescript', () => {
+    const calculator = require('./setup/calculator.ts').default;
```
Comment (ZauberNerd) [inline packages/spec/integration/jest-preset/__tests__/preset.js]: Do you need to use `.ts` here? Or could you just require `'./setup/calculator` instead?

File: packages/spec/integration/typescript/package.json [hunk 2]
```diff
@@ -9,11 +9,13 @@
     "testEnvironment": "../../helpers/env.js"
   },
   "scripts": {
-    "start": "node -e 'require(\"hops\").run(\"start\")'"
+    "start": "node -e 'require(\"hops\").run(\"start\")'",
+    "tsc": "tsc"
```
Comment (ZauberNerd) [inline packages/spec/integration/typescript/package.json]: I think renaming the script to `type-check` makes more sense.

File: packages/styled-components/package.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/xing/hops.git"
   },
   "dependencies": {
-    "babel-plugin-styled-components": "^1.5.1",
+    "babel-plugin-styled-components": "1.5.1",
```
Comment (ZauberNerd) [inline packages/styled-components/package.json]: Can we use `1.6.1` now? Or should we wait for your PR to get merged and then go to the next version?

Comment (robin-drexler) [reply packages/styled-components/package.json]: I'd wait as 1.6.1 is still rather slow

File: packages/spec/integration/typescript/package.json (hunk 2 above)
Comment (robin-drexler) [reply packages/spec/integration/typescript/package.json]: Fine with me, will adjust it in Readme as well then

**Pull request merged**